src/launchpad/size/insights/apple/loose_images.py

```diff
@@ -0,0 +1,147 @@
+        raw_image_files: list[FileInfo] = []
+        for file_info in input.file_analysis.files:
+            if self._is_loose_image_file(file_info):
+                raw_image_files.append(file_info)
+
+        if not raw_image_files:
+            return None
+
+        # Group images by canonical name
+        image_groups_dict: dict[str, list[FileInfo]] = defaultdict(list)
+        for image_file in raw_image_files:
+            canonical_name = self._get_canonical_image_name(image_file.path)
+            image_groups_dict[canonical_name].append(image_file)
+
+        image_groups = [
+            LooseImageGroup(canonical_name=canonical_name, images=images)
+            for canonical_name, images in image_groups_dict.items()
+        ]
+
+        image_groups.sort(key=lambda group: group.total_size, reverse=True)
+
+        # Calculate total savings and avoid double-counting:
+        # 1. Files eliminated via app thinning: full block-aligned disk usage saved
+        # 2. Files that remain: only block alignment waste saved
+
+        total_savings = 0
+        eliminated_files: set[str] = set()
+
+        # First pass: identify files that would be eliminated via app thinning
+        for group in image_groups:
+            eliminated_files.update(self._get_eliminated_files(group))
+
+        # Second pass: calculate savings for each file
+        for image_file in raw_image_files:
+            block_aligned_size = to_nearest_block_size(image_file.size, APPLE_FILESYSTEM_BLOCK_SIZE)
+
+            if image_file.path in eliminated_files:
+                # File eliminated via app thinning: save full block-aligned size
+                total_savings += block_aligned_size
+            else:
+                # File remains: save only block alignment waste
+                total_savings += block_aligned_size - image_file.size
+
+        return LooseImagesInsightResult(
+            image_groups=image_groups,
+            total_file_count=len(raw_image_files),
+            total_savings=total_savings,
+        )
+
+    def _is_loose_image_file(self, file_info: FileInfo) -> bool:
+        """Check if a file is a raw image that should be moved to an asset catalog."""
+        # Must be an image file
+        if file_info.file_type not in self.IMAGE_EXTENSIONS:
+            return False
+
+        # Skip if it's already in an asset catalog (.car file)
+        if ".car/" in file_info.path:
+            return False
+
+        # Skip system icons
+        filename = file_info.path.split("/")[-1]  # Get just the filename
+        if filename.startswith("AppIcon") or filename.startswith("iMessage App Icon"):
+            return False
+
+        # Skip .stickerpack directories (as mentioned in Swift code)
+        path_parts = file_info.path.split("/")
+        if any(part.endswith(".stickerpack") for part in path_parts):
+            return False
+
+        return True
+
+    def _get_canonical_image_name(self, file_path: str) -> str:
```

Comment:
This captures more than our old insight but I think it seems right after testing on some apps. The old logic required `@` in the URL.